src/topology/separation.lean

```diff
@@ -2,15 +2,92 @@
 Copyright (c) 2017 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Mario Carneiro
-
-Separation properties of topological spaces.
 -/
 import topology.subset_properties
 import topology.connected
 
+/-!
+# Separation properties of topological spaces.
+
+This file defines the predicate `separated`, and common separation axioms
+(under the Kolmogorov classification).
+
+NOTE TO REVIEWERS: not sure how to deal with the last few lemmata after line 1033
```

Comment:
I think it makes sense to add `loc_compact_t2_tot_disc_iff_tot_sep` under `T₂ spaces` and `connected_components.t2` under `If the space is also compact:`.